kube/resources/bootloader.yaml

```diff
@@ -1,40 +1,44 @@
-apiVersion: v1
-kind: Pod
+apiVersion: batch/v1
+kind: Job
```

Comment:
Kube jobs only have [best effort parallelism guarantees](https://kubernetes.io/docs/concepts/workloads/controllers/job/#handling-pod-and-container-failures), which is why I don't really like using them for crucial workflows. Took a look and confirmed this is probably the best way of doing this with Kustomize. Can we add a comment here that we generally want to use `Pod` (our Helm charts use Pod) for the best exactly-once execution guarantees and cannot do so because Kustomize does not support [`generateName`](https://github.com/kubernetes-sigs/kustomize/issues/641)? Want to prevent confusion in the future.

If Kustomize did support `generateName`, we should be able to instruct users to run `kubectl create` on initial create and `replace` on subsequent runs.

This happens relatively infrequently so risk is low. In the long term, I think we'll consolidate the Kube deploys into Helm so I think this is fine for now.